content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -457,6 +457,12 @@ Since Kubernetes 1.27, Kubelet transitions deleted pods to a terminal phase
 ensures that deleted pods have their finalizers removed by the Job controller.
 {{< /note >}}
 
+{{< note >}}
+Starting with Kubernetes v1.28, when Pod failure policy is used, the Job controller recreates
+terminating Pods only once these Pods reach the terminal `Failed` phase. This behavior is similar
+to `podRecreationPolicy: Failed`. For more information, see [Pod replacement policy](#pod-replacement-policy).
```

Comment:
```suggestion
to `podReplacementPolicy: Failed`. For more information, see [Pod replacement policy](#pod-replacement-policy).
```

there are also other instances of `podRecreationPolicy` that should be updated: https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20podRecreationPolicy&type=code

Reply:
Wow! Thanks for finding that.  I'll open up a PR today.

I'll also update the code.